### ISSUE TITLE
Using more descriptive variable names

### ISSUE THREAD

**Issue by asadmehdi785** [ssocratous/intro-data-capstone-biodiversity#3]
Firstly, nice job on running a chi square test on multiple contingencies, not just the ones that were asked of you. However, we should try to use descriptive variable names as often as possible to increase the readability of our code. Take this code for example:

https://github.com/ssocratous/intro-data-capstone-biodiversity/blob/b4b62681f63c25c067e561b782b5c5f8e73160c1/Capstone%20Option%202%20Biodiversity%20for%20the%20National%20Parks/Capstone%202%20biodiversity.py#L289-L291

You noted in a comment here that you're running a chi square test to see if the difference between Mammal and Fish is significant, which is good. However, it would be even better if we used a more descriptive variable name for the contingency table, such as `mammal_fish_contingency` for example. And similarly for the other contingency tables you created in this part.

**Comment by ssocratous** [comment]
Yes totally understand ehere you are coming from. Thanks for the feedback

**Comment by ssocratous** [comment]
*where
